contracts/mock/upgradable/UpgradableV2.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity 0.5.17;
+
+import "./UpgradableV1.sol";
+
+
+contract UpgradableV2 is UpgradableV1 {
+    function setValue(uint256 newValue) external {
```

Comment:
Right now we are validating we can change previous storage, one other case we could test is to extend storage allocation? For example V1 has 1 state variable, V2 adds 1 state variable, V3 adds 1 state variable, and we validate that V3 can change any V1, V2 and V3 variables.

Reply:
Should we add these test in your branch with the upgradeable ATM contracts?

Reply:
At the moment we don't need to change ATMGovernance state so I wouldn't add that in that branch. I think it'll be simpler to include this test on current branch because contracts are simpler (UpgradableV2, UpgradableV1)